apps/site/app/[locale]/not-found.tsx

```diff
@@ -19,13 +19,15 @@ const NotFoundPage: FC = () => {
       <main>
         404
         <h1 className="special -mt-4">{t('layouts.error.notFound.title')}</h1>
-        <div className="my-4 flex items-center justify-center">
-          <Image
-            src="/static/images/node-mascot.svg"
-            alt="The Node.js mascot"
-            height={114.69}
-            width={100}
-          />
+        <div className="my-4 flex h-[300] items-center justify-center">
+          <div className="turtle motion-safe:animate-surf motion-reduce:animate-none">
```

Comment:
this was the only way I could get the accessibility logic to work. Within CSS alone, it was not working for me.